skin.json

```diff
@@ -146,6 +146,9 @@
       "height": 81
     }
   },
+  "live": {
+    "dvrEnabled": false
```

Comment:
All HLS videos are "DVR", however it depends on how much DVR is enabled.  For example, HLS  live streams could have the bare minimum 10 seconds or 30 minutes.  How does VTC know when DVR is available?